Chess/src/chess/Main.java

```diff
@@ -679,9 +687,13 @@ else if (destinationlist.isEmpty() == false && willkingbeindanger(c, destination
                     
                     //Check for stalemate
                     if (isStalemate(chance, boardState)) {
-                    	System.out.println("Stalemate!!");
                     	triggerDraw("Stalemate");
                     }
+
+                    increaseTrivialMoveCounter();
+                    if (trivialMovesCounter >= 50){
+                        triggerDraw("50 move rule");

```

Comment:
Minor nitpick: the other draw conditions are capitalized. Please change to "50 Move Rule"